README.md

```diff
@@ -11,31 +11,29 @@
 
 ## Staging
 
-- **url**: [https://service-manual.apps.staging.digital.gov.au/](https://service-manual.apps.staging.digital.gov.au/)
-- **branch**: `develop`
-- **status**:  [![CircleCI](https://circleci.com/gh/govau/service-manual/tree/develop.svg?style=svg)](https://circleci.com/gh/govau/service-manual/tree/develop)
+- **branch**: `test-staging`
 
 
 ## Development
 
-❗️All pull requests should be compared against `develop`
```

Comment:
This is the correct comment, don't change it. Test-staging is a test branch which will likely be removed one day.

Reply:
Will fix